### PR TITLE
Add fixtures for testing phases 1 and 2 of Dockerfiles

### PR DESCRIPTION
**Pull request by natalieparellano**

### CODE REVIEW

**Comment by natalieparellano** [comment]
This should probably be blocked on https://github.com/buildpacks/pack/pull/1478 as CI won't pass until pack has been updated to support builder creation with extensions

**Comment by AidanDelaney** [comment]
Looks like we need to release a `pack` with support for `Dockerfiles` before these samples changes can land.

**Comment by natalieparellano** [comment]
@AidanDelaney the pipeline is green! WDYT?

File: Makefile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-PACK_FLAGS?=--pull-policy=never
+PULL_POLICY_NEVER?=--pull-policy=never
```
Comment (natalieparellano) [inline Makefile:1]: I have to confess, I'm not sure why this is needed.

File: Makefile [hunk 2]
```diff
@@ -28,55 +28,55 @@ build-linux-builders: build-builder-alpine build-builder-bionic
...
 	@echo "> Building 'bionic' builder..."
-	$(PACK_CMD) builder create cnbs/sample-builder:bionic --config $(SAMPLES_ROOT)/builders/bionic/builder.toml $(PACK_FLAGS)
+	$(PACK_CMD) builder create cnbs/sample-builder:bionic --config $(SAMPLES_ROOT)/builders/bionic/builder.toml $(PULL_POLICY_NEVER)
 
 build-linux-buildpacks: build-buildpacks-alpine build-buildpacks-bionic
 
 build-buildpacks-alpine: build-sample-root
 	@echo "> Creating 'hello-moon' app using 'alpine' builder..."
-	$(PACK_CMD) build sample-hello-moon-app:alpine -v --builder cnbs/sample-builder:alpine --buildpack $(SAMPLES_ROOT)/buildpacks/hello-world --buildpack $(SAMPLES_ROOT)/buildpacks/hello-moon $(PACK_FLAGS) $(PACK_BUILD_FLAGS)
+	$(PACK_CMD) build sample-hello-moon-app:alpine -v --builder cnbs/sample-builder:alpine --buildpack $(SAMPLES_ROOT)/buildpacks/hello-world --buildpack $(SAMPLES_ROOT)/buildpacks/hello-moon $(PACK_BUILD_FLAGS)
```
Comment (natalieparellano) [inline Makefile:41]: Only changed the pull policy for alpine, as that's the only builder with extensions.

Comment (AidanDelaney) [reply Makefile:41]: AFAIK it stops `pack` trying to look for `cnbs/sample-builder` on a registry.

**Review by AidanDelaney — APPROVED** [review]
PR provides an example where the image is switched to `run-image-curl`.  Also demonstrates `generate` installing the a `tree` binary via `apk`.  LGTM.

**Pull request merged**